src/lib/components/filter/components/filter-toolbar/filter-toolbar.component.ts

```diff
@@ -0,0 +1,31 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
+import { FilterChipDefDirective } from '../../directives/filter-chip-def.directive';
+import { FilterMenuDirective } from '../../directives/filter-menu.directive';
+import { Observable } from 'rxjs';
+import { MatMenu } from '@angular/material/menu';
+
+/** A toolbar providing necessary elements for applying filters to a table. */
+@Component({
+    selector: 'terra-filter-toolbar',
+    templateUrl: './filter-toolbar.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class FilterToolbarComponent {
+    /** Reference to the FilterMenuDirective containing all information needed to render the chips */
+    @Input()
+    public filterMenu: FilterMenuDirective;
+
+    /** Fires when the user clicks the search button in the filter toolbar */
+    @Output()
+    public search: EventEmitter<void> = new EventEmitter();
+
```

Comment:
Habe in der Component noch eine ViewChild Variable hinzugefügt, um von der neuen no-result-component an den Filter zu kommen:
https://github.com/plentymarkets/terra/pull/6306/files#diff-af390c2653c00991d35ebc2d4bd4c210c01c675a5ec8fe6d938a58774c5a1c41R21-R24

Vllt übernimmst du das schonmal. Sollte keine Auswirkungen haben